<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Image/main.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="deviceListWidget">
         <property name="minimumSize">
          <size>
           <width>630</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>70</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="selectALLOrNotCheckBox">
            <property name="text">
             <string>全选</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>71</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="informationPushBtn">
            <property name="text">
             <string>信息操作</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>上传</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mediaDownloadPushBtn">
            <property name="text">
             <string>介质定义下载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="operatorDownloadPushBtn">
            <property name="text">
             <string>用户定义下载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="autoConfigBtn">
            <property name="text">
             <string>自动化配置</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>70</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="infoWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="operationWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统管理</string>
    </property>
    <addaction name="actionAutoConfigOp"/>
    <addaction name="actionExit"/>
    <addaction name="paraConfigAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>加卸载操作</string>
    </property>
    <addaction name="actionFindOp"/>
    <addaction name="actionInformationOp"/>
    <addaction name="actionUploadOp"/>
    <addaction name="actionMediaDownloadOp"/>
    <addaction name="actionOperatorDownloadOp"/>
    <addaction name="actionAbortOp"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionMainWindow"/>
    <addaction name="actionSysLog"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionToolVersion"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFindOp"/>
   <addaction name="actionInformationOp"/>
   <addaction name="actionUploadOp"/>
   <addaction name="actionMediaDownloadOp"/>
   <addaction name="actionOperatorDownloadOp"/>
  </widget>
  <widget class="QDockWidget" name="logDockWidget">
   <property name="minimumSize">
    <size>
     <width>109</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>操作日志</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextBrowser" name="logTextBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAutoConfigOp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/autoConfig.png</normaloff>:/Image/autoConfig.png</iconset>
   </property>
   <property name="text">
    <string>自动化配置</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/exit.png</normaloff>:/Image/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionFindOp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/find.png</normaloff>:/Image/find.png</iconset>
   </property>
   <property name="text">
    <string>设备发现</string>
   </property>
  </action>
  <action name="actionInformationOp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/Information.png</normaloff>:/Image/Information.png</iconset>
   </property>
   <property name="text">
    <string>设备信息</string>
   </property>
  </action>
  <action name="actionUploadOp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/upload.png</normaloff>:/Image/upload.png</iconset>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </action>
  <action name="actionMediaDownloadOp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/download1.png</normaloff>:/Image/download1.png</iconset>
   </property>
   <property name="text">
    <string>介质定义下载</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOperatorDownloadOp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/download2.png</normaloff>:/Image/download2.png</iconset>
   </property>
   <property name="text">
    <string>操作者定义下载</string>
   </property>
  </action>
  <action name="actionMainWindow">
   <property name="text">
    <string>主界面</string>
   </property>
  </action>
  <action name="actionSysLog">
   <property name="text">
    <string>系统日志</string>
   </property>
  </action>
  <action name="actionToolVersion">
   <property name="text">
    <string>工具说明</string>
   </property>
  </action>
  <action name="actionAbortOp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>AbortOp</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>自动化配置</string>
   </property>
  </action>
  <action name="paraConfigAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Image/setting.png</normaloff>:/Image/setting.png</iconset>
   </property>
   <property name="text">
    <string>传输参数配置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
